CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ cmake_policy(SET CMP0074 NEW)
 
 # Project
 project(onnx C CXX)
-option(ONNX_USE_PROTOBUF_SHARED_LIBS "Build ONNX using protobuf shared library. Sets PROTOBUF_USE_DLLS CMAKE Flag and Protobuf_USE_STATIC_LIBS. " OFF)
+option(ONNX_USE_PROTOBUF_SHARED_LIBS "Build ONNX using protobuf shared library." OFF)
```

Comment:
More information is usually better. Why remove it?

Reply:
These are CMake variables for finding Protobuf, they are package finding details and should not be exposed and set by the user. For example, if the user chooses to build a static ONNX lib and set the PROTOBUF_USE_DLLS macro, there may be linking error.